test/topology_custom/test_tablets.py

```diff
@@ -144,6 +146,28 @@ async def check_allocated_replica(expected: int):
     await check_allocated_replica(rf_to)
 
 
+@pytest.mark.asyncio
+@pytest.mark.xfail(reason="Issue #18786, reading from mutation fragments crashes the node")
+async def test_tablet_mutation_fragments_sanity(manager: ManagerClient):
```

Comment:
As usual, I wish people will add comments to test explaining *why* they check what they check, to future readers who will look at these tests perplexed and have no idea why they check this specific thing, and refer to the issue they reproduce (not just in the "xfail" string that you'll remove once the issue is fixed).

If I understand correctly, the main intention of this test was to have more nodes than RF (3>2) so that a node doesn't have copies of all tablets, and this confused MUTATION_FRAGMENT. Or something.

Reply:
Isn't "reading from mutation fragments crashes the node" explanatory enough?

Reply:
But this string is not a comment - it's an xfail reason, which you plan to delete as soon as the bug is fixed :-( I'd still like to have a comment including a "Reproduces ..." in this comment even after the bug is fixed.

But moreover, no, "reading from mutation fragments crashes the node" is probably not self-explanatory enough... I had a specific question: It looks like you went out of your way to use RF (2) which is lower than the number of nodes (3). Why? Is this an important part of making it crash? If you used RF=N,  is this crash gone? This is exactly what I would have liked to see explained.